benchmarks/benchmarks_stats.py

```diff
@@ -0,0 +1,34 @@
+"""Benchmarks suite for stats module."""
+
+import numpy as np
+import xarray as xr
+
+from sgkit import count_call_alleles, count_cohort_alleles
+from sgkit.tests.test_aggregation import get_dataset
+
+
+def time_count_call_alleles() -> None:
+    count_call_alleles(
+        get_dataset(
```

Comment:
I don't think the dataset creation should be a part of the benchmark, and should be moved to `setup` function, see https://asv.readthedocs.io/en/stable/writing_benchmarks.html#timing

Reply:
Makes sense.